### DISCUSSION TITLE
Smtp error - Mozilla Hubs login failure

### DISCUSSION THREAD

**Discussion by tksham** [mozilla/hubs#5481] [Hubs Cloud Questions]
We are using the Hubs Cloud Enterprise stack from AWS, and we previously was using the smtp setting to send out the magic link for login.

Somehow the smtp no longer works and we can't login to fine-tune that setting.

Is there any other way that we can force login without the email magic link? 
Even we want to go back to use SES but we can't as we cannot login.

**Comment by pattersonbl2** [comment]
Are you able to ssh into your instance to change some configs for Smtp ?

**Comment by tksham** [comment]
No, I can't. I am new to AWS, is there any tutorial for resetting it via ssh?
tried ssh -i "xxx.pem" root@ec.bla.bla..compute.amazonaws.com or ssh -i "xxx.pem" root@public-ip
Have a Permission denied (keyboard-interactive) error prompt.

Does the ssh closed in some default setting from Hubs Cloud?

**Comment by pattersonbl2** [comment]
You should ssh using 
ssh -i <key file> ubuntu@<server name>.<website address> 

The server name would be the server's actual name in the EC2 console.

**Comment by tksham** [comment]
thanks! from EC2 console it said root ^^"
but it now prompts me to input verification code (I have used my ssh key already and I don't know where I can get that verification code)

**Reply by pattersonbl2** [reply]
Did you set up the 2-factor authentication when you first started your instance?   if not you can find the code to add it manually to the google auth app in the S3 bucket named box-keys.  You should be able to download a file called ssh-totp.cfg with it in it.

**Reply by tksham** [reply]
thanks!! you are really helpful, is there any guide for commands with ssh, like resetting the smtp setting?
